src/main/java/CMV.java

```diff
@@ -1,3 +1,5 @@
+import javax.swing.*;
```

Comment:
I'm unsure why swing is imported here. Remove this line

Reply:
Agreed, might unknowingly pull in a dependency